recipes/naturalneighbor/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "naturalneighbor" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/naturalneighbor-{{ version }}.tar.gz
+  sha256: 647b5922c3787887c04be7748f8112a1720b1323c8e86b4c6665ce7eee89aee7
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
```

Comment:
```suggestion
  skip: True  # [py<312]
  number: 0
```

Reply:
It didn't work - stage-recipes check still builds the package for python versions 3.12. However, upon explicitly mentioning the supported python versions in `conda_build_config.yaml` python 3.12 is ignored.

Reply:
It worked now! I accidentally used `skip: True  # [py<312]`  in the `meta.yaml` instead of `skip: True  # [py>=312]`. Please accept my apologies for the trouble.